radicle-cli/src/main.rs

```diff
@@ -62,6 +62,9 @@ fn parse_args() -> anyhow::Result<Command> {
     Ok(command.unwrap_or_else(|| Command::Other(vec![])))
 }
 
+/// Print radicle client's version.
+///
+/// Third party applications use it to parse Radicle Cli's version.
```

Comment:
Anything in particular to test?  The direct method of testing expected output wont give us much, just a rewrite of `print_version`.

Reply:
I would test that it matches the pattern `radicle vX.X.X (<hex>)` maybe?

I don't think it's important, but neither do I think the comment will help.. It's on a private function.

Reply:
> I would test that it matches the pattern radicle vX.X.X (<hex>) maybe?

Agreed, I'll do this

Reply:
how about moving closer to semver ?

sounds like the `hex` could be added as either a pre-release version (https://semver.org/#spec-item-9), or build metadata (https://semver.org/#spec-item-10), no ?

Reply:
@gsaslis we're doing this for '-dev' builds i.e. deployments build from the repository directly, and not via releases.  I'm unsure there's a point for regular releases, because version at that point should always point to the given commit.